### ISSUE TITLE
Training the model

### ISSUE THREAD

**Issue by wzk101** [sohailahmedkhan/Video_Transformer_Deepfake_Detector#2]
2024-03-21 21:08:57.406070850 [W:onnxruntime:, execution_frame.cc:806 VerifyOutputSizes] Expected shape from model of {1,16680,2} does not match actual shape of {1,2134,2} for output 353
2024-03-21 21:08:57.406470419 [W:onnxruntime:, execution_frame.cc:806 VerifyOutputSizes] Expected shape from model of {output_dynamic_axes_1,16680,4} does not match actual shape of {1,2134,4} for output output
2024-03-21 21:08:57.573030034 [W:onnxruntime:, execution_frame.cc:806 VerifyOutputSizes] Expected shape from model of {1,16680,2} does not match actual shape of {1,2134,2} for output 353
2024-03-21 21:08:57.573486190 [W:onnxruntime:, execution_frame.cc:806 VerifyOutputSizes] Expected shape from model of {output_dynamic_axes_1,16680,4} does not match actual shape of {1,2134,4} for output output' 

 when I run the train_image_ViT.py will raise this error

**Comment by sohailahmedkhan** [comment]
Hi. It seems something is wrong with the inputs that you are providing. Please double check the input dimensions of images. If you still get the error, then please provide the full error trace.

**Comment by wzk101** [comment]
The inputs are 3*299*299, following the setting in your essay, isn't it correct?  And it seems the warning didn't block code running.  
However, it may meet some problem about memery.   
"epoch_accuracy += acc/ len(train_loader)
        epoch_loss += loss / len(train_loader)" 
Memory footprint gets bigger during training, and I change it into:
 epoch_accuracy += acc.item() / len(train_loader)
        epoch_loss += loss.item() / len(train_loader)

**Comment by sohailahmedkhan** [comment]
Yes, you are right. The input image size is 3 x 299 x 299. If you still face any errors, I would suggest you paste the whole error trace that you get when you run the train file.
